configs/ci/roles/subscriptions.json

```diff
@@ -2,10 +2,11 @@
   "roles": [
     {
       "name": "Subscription Watch administrator",
-      "description": "Perform any available operation against any Subscription Watch resource.",
+      "display_name": "Subscriptions administrator",
+      "description": "Perform any available operation against any Subscriptions resource.",
       "system": true,
-      "platform_default": true,
-      "version": 4,
+      "platform_default": false,
+      "version": 5,
       "access": [
         {
           "permission": "subscriptions:*:*"
```

Comment:
Hey, @patrickeasters - based on the email thread with Nicole, is the admin role meant to have `subscriptions:manifests:write` now as well?

Reply:
Yea, it's intended to have that now via the implicit `subscriptions:*:*`

Reply:
👍